vault/resource_identity_entity_alias.go

```diff
@@ -63,6 +63,17 @@ func identityEntityAliasCreate(d *schema.ResourceData, meta interface{}) error {
 	if err != nil {
 		return fmt.Errorf("error writing IdentityEntityAlias to %q: %s", name, err)
 	}
+
+	if resp == nil {
+		aliasIDMsg := "Unable to determine alias id."
+
+		if aliasID, err := findAliasID(client, canonicalID, name, mountAccessor); err == nil {
+			aliasIDMsg = fmt.Sprintf("Alias resource ID %q may be imported.", aliasID)
+		}
+
+		return fmt.Errorf("IdentityEntityAlias %q already exists. %s", name, aliasIDMsg)
```

Comment:
This is interesting, I previously thought all errors in Go should be lowercase, but I see [here](https://github.com/golang/go/wiki/CodeReviewComments#error-strings) that if they're full strings (like these are), it's okay sometimes to capitalize them.